packages/core/src/awsService/appBuilder/explorer/nodes/deployedNode.ts

```
@@ -101,31 +97,6 @@ export async function generateDeployedNode(
                         code: 'lambdaClientError',
                     })
                 }
-                const connection = await getIAMConnection({ prompt: false })
-                if (!connection || connection.type !== 'iam') {
-                    return [
-                        createPlaceholderItem(
-                            localize(
-                                'AWS.appBuilder.explorerNode.unavailableDeployedResource',
-                                '[Failed to retrive deployed resource.]'
-                            )
-                        ),
-                    ]
-                }
```

Why are we removing this connection check?

---

All this block of code is to get the Log Group name using SDK v3. 

In theory we could keep this check for IAM connection, but the connection was only being used to set up the v3 client in the following lines, so in reality we don't really need to check for anything because we're not making any other call.

If there's a failure it should throw the error on the lines before this.